src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs

```diff
@@ -61,10 +61,7 @@ public void TearDown()
             channelProvider?.Dispose();
         }
 
-        protected bool TryWaitForMessageReceipt()
-        {
-            return TryReceiveMessage(out var message, incomingMessageTimeout);
-        }
+        protected bool TryWaitForMessageReceipt() => TryReceiveMessage(out var message, incomingMessageTimeout);
```

Comment:
👍 I like `_` for unused vars. I'll make the change (will amend directly since the change set is so trivial).